ajax/libs/leaflet-polylinedecorator/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "leaflet-polylinedecorator",
+  "filename": "leaflet.polylineDecorator.min.js",
+  "npmName": "leaflet-polylinedecorator",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "leaflet-polylinedecorator*"
+      ]
+    }
+  ],
+  "version": "1.0.1",
+  "authors": [
+    "Benjamin",
+    "Becquet"
```

Comment:
No, I was wrong.
So I want to ask whether the correct form is: ```"author": "Benjamin Becquet"``` or not?
Because there is just one author.
Thanks.